web/account_balance_controller_test.go

```diff
@@ -49,3 +55,47 @@ func TestAccountBalanceController_Index(t *testing.T) {
 	assert.Equal(t, "0.000000000000000256", ab.EthBalance.String())
 	assert.Equal(t, "0.000000000000000256", ab.LinkBalance.String())
 }
+
+func TestAccountBalanceController_Withdraw(t *testing.T) {
+	appWithAccount, cleanup := cltest.NewApplicationWithKeyStore()
+	defer cleanup()
+
+	ethMock := appWithAccount.MockEthClient()
+	sentAt := "0x5BA0"
+	hash := cltest.NewHash()
```

Comment:
This is used later in the response to `sendRawTransaction` but that is thrown away, doesn't really matter here. The easiest way to know what the transaction hash is going to be is to set up an application with [cltest.NewConfigWithPrivateKey](https://github.com/smartcontractkit/chainlink/blob/master/integration/features_test.go#L48-L49). Once you have that, you should get the same expected value every time.

Reply:
```
	config, _ := cltest.NewConfigWithPrivateKey()
	app, cleanup := cltest.NewApplicationWithConfigAndKeyStore(config)
```